src/data/d2/powerful-rewards.json

```diff
@@ -1,21 +1,5 @@
 [
-  66825403,
-  73143230,
-  226330008,
-  506041342,
-  646467119,
-  782362349,
-  914894006,
-  993006552,
-  1204101093,
-  2223145359,
-  2246571627,
-  2484791497,
-  2646629159,
-  3114385605,
-  3114385606,
-  3114385607,
-  3308857600,
-  3529617614,
-  3879752304
+  66825403, 73143230, 226330008, 506041342, 646467119, 782362349, 914894006, 993006552, 1204101093,
+  2223145359, 2246571627, 2484791497, 2646629159, 3114385605, 3114385606, 3114385607, 3308857600,
+  3529617614, 3879752304
```

Comment:
no

Reply:
This is that bloody prettier git hook doing this. I didn't go anywhere near this file. It drives me nuts